src/xdocs/config.xml

```diff
@@ -38,7 +38,7 @@
         <li><em>Filters</em>
 
         - modules that filter audit events,
-        including error messages, for acceptance.</li>
+        including messages, for acceptance.</li>
```

Comment:
Should this be `violation messages`?

Reply:
no as "audit event" is specified, is better to not put its synonym.